### ISSUE TITLE
some problems during the training process

### ISSUE THREAD

**Issue by WidCoder** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#312]
Hi,dear author, I used your project for testing and it was great. However, during my training, I encountered the following issues.I caught the following exception by modifying the val.py code. Through the terminal's output, I found that the attribute of (normed_img) was somewhat empty, but I am not sure what caused these exceptions. I hope you can help me solve this problem. Thanks

![image](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/assets/87012220/722ed9dd-da44-4be2-8a38-5b57a770678f)
![QQ截图20230515173057](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/assets/87012220/18a9827a-d26d-414a-86db-7cf0bf67d273)

**Comment by Daniil-Osokin** [comment]
Hi! I believe you should not skip the image. If the image is empty (likely wrong path to the image in data loader), just delete it from your annotation file.

**Comment by WidCoder** [comment]
Thank you very much for your reply. Through the method you mentioned, I have solved the problem, but I found that the training time for the model is a bit long. Is there any way to speed up the training time? I tried to modify the batch size before, but the improvement was not particularly significant. I trained 25000 Iters, which took about five hours

**Comment by Daniil-Osokin** [comment]
For faster training you can decrease network input size, modify [CropPad](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/blob/master/train.py#LL37C36-L37C43) parameter. However this might affect final accuracy.
